lms/djangoapps/commerce/views.py

```diff
@@ -111,7 +111,8 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
 
         try:
             timeout = getattr(settings, 'ECOMMERCE_API_TIMEOUT', 5)
-            response = requests.post(url, data={'sku': course_modes[0].sku}, headers=headers, timeout=timeout)
+            response = requests.post(url, data=json.dumps({'sku': course_modes[0].sku}), headers=headers,
```

Comment:
I'm sorry I didn't catch this earlier, but why are we assuming that it's the first element in `course_mode`? I know we're going to be changing things later, but this seems fragile.